pkgs/tools/security/rnp/default.nix

```diff
@@ -0,0 +1,46 @@
+{ lib
+, stdenv
+, botan2
+, bzip2
+, cmake
+, fetchurl
+, json_c
+, unzip
+, zlib
+}:
+let
+  version = "0.14.0";
+  name = "rnp-${version}";
+in
```

Comment:
Can you move the `pname` and `version` inside `stdenv.mkDerivation` instead of using a `let ... in`